Lib/test/test_ordered_dict.py

```diff
@@ -650,6 +650,41 @@ def test_free_after_iterating(self):
         support.check_free_after_iterating(self, lambda d: iter(d.values()), self.OrderedDict)
         support.check_free_after_iterating(self, lambda d: iter(d.items()), self.OrderedDict)
 
+    # TODO: This test is relating to PEP 468.
+    # Move this test to somewhere more appropriate.
+    def test_kwargs_order(self):
+        # bpo-34320
+        od = self.OrderedDict([('a', 1), ('b', 2)])
+        od.move_to_end('a')
+        expected = list(od.items())
+
+        def fn(**kw):
+            return kw
+
+        res = fn(**od)
+        self.assertIsInstance(res, dict)
+        self.assertEqual(list(res.items()), expected)
+
+    # TODO: This test is relating to PEP 520.
```

Comment:
This test is not specific to PEP 520, which only relates directly to the class definition namespace.  So you should drop this comment.